src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.c

```diff
@@ -13,8 +13,17 @@
 void ItemEtcetera_Init(Actor* thisx, GlobalContext* globalCtx);
 void ItemEtcetera_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void ItemEtcetera_Update(Actor* thisx, GlobalContext* globalCtx);
+void func_80B85C64(Actor* thisx, GlobalContext* globalCtx);
+void ItemEtcetera_DrawFireArrow(Actor* thisx, GlobalContext* globalCtx);
```

Comment:
commented on this in another pr, but imo separate the og main funcs from the extra ones

Reply:
disregard